xikolo-ios/ViewControllers/Courses/CourseContentTableViewController.swift

```diff
@@ -139,13 +114,10 @@ class CourseContentTableViewController: UITableViewController {
         }
     }
 
-    func reachabilityChanged(_ note: Notification) {
-        guard let reachability = note.object as? Reachability else { return }
-
-        let oldOfflinesState = self.isOffline
-        self.isOffline = !reachability.isReachable
-
-        if oldOfflinesState != self.isOffline {
+    func reachabilityChanged() {
+        let oldOfflineState = self.isOffline
+        self.isOffline = ReachabilityHelper.isOffline
+        if oldOfflineState != self.isOffline {
```

Comment:
This can be moved into the `didSet` block of `isOffline`. Here you can use `oldValue` (see Property Observers section in https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Properties.html#//apple_ref/doc/uid/TP40014097-CH14-ID254)